modules/nw-dns-forward.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * dns/dns-operator.adoc
+
+[id="nw-dns-forward_{context}"]
+= Using DNS forwarding on user-provisioned infrastructure in AWS
```

Comment:
This is not specific to AWS.

Reply:
Thanks, @Miciah! I leaned on the enhancement doc to doc DNS forwarding, but it seems like it was out of date. Updates are coming.